default.nix

```diff
@@ -21,8 +21,8 @@ let
 in with pkgs; {
   ghc = haskellPackages.aeson-combinators;
   ghcjs = ghcjs.pkgs.haskell.packages.ghcjs.callCabal2nix "aeson-combinators" ./. {};
-  ghc865 = haskell.packages.ghc865.aeson-combinators;
-  ghc883 = haskell.packages.ghc883.aeson-combinators;
   ghc884 = haskell.packages.ghc884.aeson-combinators;
-  ghc8101 = haskell.packages.ghc8101.aeson-combinators;
+  ghc8107 = haskell.packages.ghc8107.aeson-combinators;
+  ghc902 = haskell.packages.ghc902.aeson-combinators;
+  ghc921 = haskell.packages.ghc921.aeson-combinators;
```

Comment:
for some reason even when using these 2 attributes ghc-pkg says I'm using `aeson-1.5.6.0`

Reply:
At least on my nightly:
```
> haskell.packages.ghc921.aeson.version
"1.5.6.0"
```

So, nix is still going to use 1.5.6.0. I can imagine that stackage didn't manage to migrate to aeson 2.0.0.0 yet.